addons/website_sale/static/src/website_builder/products_item_option_plugin.js

```diff
@@ -19,12 +19,11 @@ class ProductsItemOptionPlugin extends Plugin {
         "addRibbon",
         "getRibbons",
         "_deleteRibbon",
-        "_setRibbon"
+        "_setRibbon",
```

Comment:
maybe remove the _ . It s strange to share something that seems private. (Not really link to your PR)